### ISSUE TITLE
Update naming conventions 

### ISSUE THREAD

**Issue by RobStallion** [dwyl/autoform#27]
Rename `render_autoform` and `custom_render_autoform` functions so we have a more consistent naming convention. 

@nelsonic makes a good point [here](https://github.com/dwyl/autoform/pull/22) about naming convention.

**Comment by RobStallion** [comment]
@nelsonic I don't want to simply change the names of the functions as it could potentially break projects that are currently using this module.

It should be okay as they should be on a version that will still have the functions named this way but I think it's best to be on the safe side. 

I was planning to create 2 new functions called **`autoform_render/4`** and **`autoform_render_custom/4`** and having them call the old functions passing the arguments in. Then updating the docs and examples to only show uses of the newer naming convention.

This was no projects using `autoform` will be effected but everyone who uses it going forward will only see documentation about the newly named functions.

Do you have any thoughts on that approach? 

(Also I think that we might be able to combine the 2 functions into one so we eventually just end up with `autoform_render`. @Danwhy do you have any thoughts on this? @nelsonic @Danwhy Would this be a good time to tackle both points?)

**Comment by Danwhy** [comment]
Renaming the functions, but keeping the old ones (marked as deprecated) that just forward the call to the new name seems like a good idea to me.

Also agree that having two different functions does seem a little unnecessary as the custom one could easily act as the default, so it might be nice to combine them.

**Comment by RobStallion** [comment]
@Danwhy Good point about marking them as depricated 👍

I just had a thought about a possible simple way to achieve the combining of the functions as well. With the solution above we could make it seem as the though functions have been combined when in reality the newly created **autoform_render** could just handle the logic of which function to call based on the arguments given to it. @Danwhy Do you think this could work?

**Comment by Danwhy** [comment]
@robstallion Yeah that should work great for now, then we can work on refactoring them later

**Comment by RobStallion** [comment]
@Danwhy I have started a branch for these changes [here](https://github.com/dwyl/autoform/pull/28). At the moment it is very basic but I just wanted your thoughts on the approach before taking it any further. 

If you could take a look at the pr and let me know if you have any thoughts on it so far that would be greatly appreciated 👍

**Comment by RobStallion** [comment]
@Danwhy @nelsonic do you have any thoughts/comments/suggestions on this approach?
